### DISCUSSION TITLE
Testers for Google Play

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#57] [General]
This is sad news for those who waiting for Google Play releases.

For new developer accounts (such as [Fossify](https://play.google.com/store/apps/dev?id=7297838378654322558)), it is now a requirement to have at least 20 testers opt-in on a closed track before the apps can be published on the production track. We have to run closed tests for at least 14 days before applying for production.

If you are interested in helping out with this, please share your email addresses. You'll be added as a tester on the Fossify Google Play Developer account. When you get an email, please install the app(s).  If you want to avoid sharing your email address publicly, send it to hello@fossify.org

A perk of being a tester: You will get access to any potential pro versions and Fossify Thank You without being charged. This is a Google Play Store feature.

**Comment by inson1** [comment]
@naveensingh 
1. Where is email ID? Do you mean email address?
2. Can we keep the apps? Or would be some special edition, that would be deleted afer testing and needed reinstaling? Does somebody have experience with that?
3. How show we test them? Or we dont have to?

**Reply by naveensingh** [reply]
> @naveensingh 
> 1. Where is email ID? Do you mean email address?
> 2. Can we keep the apps? Or would be some special edition, that would be deleted afer testing and needed reinstaling? Does somebody have experience with that?
> 3. How show we test them? Or we dont have to?

1. Yes, I meant email address.
2. Yes, you can. It'll be updated with the public version but you might still see a tester warning.
3. I think installation is enough, they probably do track how much the app is used over the 14 day period internally. Just use it like you would use the normal version.

**Reply by inson1** [reply]
@naveensingh email sent

**Reply by inson1** [reply]
@naveensingh PS Im not active on that email, ping me here if anything comes up

**Comment by MrCamel999** [comment]
I would like to help out, but I want to install the fdroid versions whenever they release on my primary phone here. I have an old phone though, do you think I could use that to help get the count up without messing with my primary phone's setup?

**Reply by naveensingh** [reply]
@MrCamel999 Yes, you can use your old phone with a Google account. Thanks

**Comment by pokegamer5547** [comment]
Hello sir. um i decided to send my email to your email address instead. Could you please check and add me as the tester too? I would love to contribute in way possible. Thankyou.

**Reply by naveensingh** [reply]
I got it :)

**Comment by technodrk** [comment]
I sent my e-mail address naveensingh

**Comment by Cwpute** [comment]
Do you need any more testers ?

**Reply by inson1** [reply]
@Cwpute I think yes, he needs 20 people

**Comment by compuguy** [comment]
Sent an email to the email address specified by @naveensingh.

**Comment by DwainZwerg** [comment]
@naveensingh Do you think, you could add GitHub-APK-Releases to the Apps in the "Releases" tab, so one can easily update Fossifyapps per Obtainium once the GooglePlay tests are through?

**Reply by Aga-C** [reply]
@DwainZwerg Gallery is already released like that. Other apps APKs will also be available in Releases.

**Comment by ronniedroid** [comment]
ronnie.nissan@proton.me 

Add me as a tester.

**Reply by naveensingh** [reply]
@ronniedroid Google's system says this email does not exist. Maybe you have a gmail?

**Reply by ronniedroid** [reply]
ronnie.n.yousif@gmail.com

**Reply by ronniedroid** [reply]
@naveensingh have you added me as a tester? I am not able to reach the apps with the link.

Could it be because I am from Iraq?

**Reply by naveensingh** [reply]
@ronniedroid Yes. 

You must select the account associated with [ronnie.n.yousif@gmail.com](mailto:ronnie.n.yousif@gmail.com) in the Google Play app for the apps to show up.

> Could it be because I am from Iraq?

No, the test versions are available worldwide.

**Reply by ronniedroid** [reply]
> You must select the account associated with [ronnie.n.yousif@gmail.com](mailto:ronnie.n.yousif@gmail.com) in the Google Play app for the apps to show up.
>

@naveensingh I have selected the account, and it still does not show up.

**Reply by naveensingh** [reply]
Can you try joining on the web: https://play.google.com/apps/testing/org.fossify.calendar

**Reply by ronniedroid** [reply]
Still can't

>App not available
>A testing version of this app hasn't been published yet or isn't available for this account.
>
>If you've been invited to become a tester, make sure you're signed in to the account that was invited to the testing program. If you've been invited to a Google Group as part of the program, make sure you've joined the Group.

**Reply by naveensingh** [reply]
@ronniedroid I re-added your email, please check if it works now.

**Reply by ronniedroid** [reply]
@naveensingh still doesn't work. 

Don't worry about it, guess we do have enough testers already, right?

**Reply by naveensingh** [reply]
For the currently released apps, yes we do. I have a feeling the less popular will have a hard time when they are released.

**Reply by ronniedroid** [reply]
> For the currently released apps, yes we do. I have a feeling the less popular will have a hard time when they are released.

Then I will try again for the less popular 🙂
Maybe get the family members on it too 😅

**Comment by claydoh** [comment]
I have sent my email to @naveensingh. I am willing to test this.

**Comment by Marechan0** [comment]
Hello, would love to help but will a phone without Google Play Services be fit?
(Through AuroraStore with a logged Google account)

**Reply by inson1** [reply]
send him the email, worst case scenario is that it wouldnt work imo
probably it will not work, but lets try it

**Reply by naveensingh** [reply]
@Marechan0 not sure if it will work, I'll add it anyway.

**Comment by inson1** [comment]
@naveensingh when it starts?

**Reply by naveensingh** [reply]
@inson1 still in review...

**Comment by Niko4u5** [comment]
I send you an Email

**Comment by GitCurious** [comment]
Email sent - will be supporting on G Play when apps are available.

**Comment by naveensingh** [comment]
I have received a lot of emails and I want to thank everyone willing to test the Fossify apps.

**Fossify Calendar** is now available for testing on Google Play. Here are the links:

Join on the web: https://play.google.com/apps/testing/org.fossify.calendar
Google Play link: https://play.google.com/store/apps/details?id=org.fossify.calendar

I have added all the emails I have received to the testers list so you **must be logged in using the proper Google account** for the app to show up.

**Comment by naveensingh** [comment]
Fossify Gallery is still in review...

**Reply by inson1** [reply]
@naveensingh Its because of the permissons or Why does it take longer?

**Reply by naveensingh** [reply]
@inson1 each app is thoroughly reviewed by Google to ensure it doesn't contain anything it shouldn't. I updated the Gallery after fixing some bugs and removing the panorama library thus delaying the review process.

**Reply by nanny07** [reply]
If you need any other tester for the Galley, I'll send you the mail

**Reply by inson1** [reply]
@nanny07 send it, gallery isnt yet out on google play, https://github.com/orgs/FossifyOrg/discussions/57#discussioncomment-8087494

**Comment by naveensingh** [comment]
If you find any issues, please report them here or in the app's repository.

**Comment by BitRanger** [comment]
I sent my email

**Comment by djal1e** [comment]
Adresse mail envoyé.
Thanks

**Reply by naveensingh** [reply]
@djal1e Ajouté à la liste des testeurs.

**Comment by naveensingh** [comment]
**Fossify File Manager** is now available for testing:

**Option 1: Join directly from your Android device:**

- Tap this link on your phone or tablet: https://play.google.com/store/apps/details?id=org.fossify.filemanager
- Install the app and start exploring!

**Option 2: Join via the web:**

-  Click this link: https://play.google.com/apps/testing/org.fossify.filemanager
-  Sign in with the same Google account you use on your Android device.
-  Click the "Become a Tester" button.

Thanks!

**Reply by inson1** [reply]
@naveensingh Do you have some statistics? like number of testers, time spend, time needed etc?

**Reply by naveensingh** [reply]
I have added 87 emails to the tester list. We probably need more people because not everyone is going to install every app.

Calendar already has 20+ installs. Assume it'll take 15-20 days from the date of the initial release before we can do open testing.

**Reply by inson1** [reply]
87 emails, crazy

**Comment by nehemiagurl** [comment]
Will Gallery be added to Google Play testers soon as well? Right now there's no apps under org.fossify.gallery, even when accessing directly via the testing url.

**Reply by naveensingh** [reply]
@nehemiagurl Gallery is still being reviewed by the Google Play team and unless they find some problem (e.g. policy violation), it should be available soon.

**Reply by nehemiagurl** [reply]
cool, thanks!

**Comment by bryonadams** [comment]
Email sent to hello@fossify.org, happy to help test the apps out if it's still needed.

**Reply by himanshu-mahendra-jain** [reply]
@naveensingh Hi, email sent. Please add me to the testing.

**Reply by naveensingh** [reply]
@hj-collab okay :)

**Reply by bryonadams** [reply]
Hi @naveensingh I'm not sure if we don't need more or if Outlook.com is being silly. Is there another address I can try to send to? I haven't heard back yet. Understood if you're busy :)

**Reply by naveensingh** [reply]
@bryonadams Sorry, it's been added already. Please follow the instructions [above](https://github.com/orgs/FossifyOrg/discussions/57#discussioncomment-8086296) to install the apps.

I may or may not respond to emails regarding testing as there are many but everyone will be notified when a new app is available.

**Comment by naveensingh** [comment]
## Tracking Google Play Availability:

|      App Name      | Closed Track | Open Track | Production |                                    Download Link                                    |
|:------------------:|:------------:|:----------:|:----------:|:-----------------------------------------------------------------------------------:|
|  **App Drawer**  |      (cancelled)       |     (cancelled)      |     (cancelled)      |   (cancelled)   |
|   **Calculator**   |      [✔️](https://play.google.com/apps/testing/org.fossify.math)       |     ✔️      |     ❌      |  [Download](https://play.google.com/store/apps/details?id=org.fossify.math)   |
|    **Calendar**    |      [✔️](https://play.google.com/apps/testing/org.fossify.calendar)      |     ✔️      |     ✔️      |   [Download](https://play.google.com/store/apps/details?id=org.fossify.calendar)    |
|     **Camera**     |      [✔️](https://play.google.com/apps/testing/org.fossify.camera)       |     ✔️      |     ❌      |    [Download](https://play.google.com/store/apps/details?id=org.fossify.camera)     |
|     **Clock**      |      [✔️](https://play.google.com/apps/testing/org.fossify.clock)       |     ✔️      |     ❌      |     [Download](https://play.google.com/store/apps/details?id=org.fossify.clock)     |
|    **Contacts**    |      [✔️](https://play.google.com/apps/testing/org.fossify.contacts)       |     ✔️      |     ✔️      |   [Download](https://play.google.com/store/apps/details?id=org.fossify.contacts)    |
|     **Phone**      |      [✔️](https://play.google.com/apps/testing/org.fossify.phone)      |     ✔️      |     ✔️      |     [Download](https://play.google.com/store/apps/details?id=org.fossify.phone)     |
|  **File Manager**  |     [ ✔️](https://play.google.com/apps/testing/org.fossify.filemanager)      |     ✔️      |     ✔️      |  [Download](https://play.google.com/store/apps/details?id=org.fossify.filemanager)  |
|   **Flashlight**   |      ❌       |     ❌      |     ❌      |  [Download](https://play.google.com/store/apps/details?id=org.fossify.flashlight)   |
|    **Gallery**     |      [✔️](https://play.google.com/apps/testing/org.fossify.gallery)       |      ✔️      |     ✔️      |    [Download](https://play.google.com/store/apps/details?id=org.fossify.gallery)    |
|    **Keyboard**    |      [✔️](https://play.google.com/apps/testing/org.fossify.keyboard)       |     ✔️      |     ❌      |   [Download](https://play.google.com/store/apps/details?id=org.fossify.keyboard)    |
|    **Launcher**    |      [✔️](https://play.google.com/apps/testing/org.fossify.home)       |     ❌      |     ❌      |   [Download](https://play.google.com/store/apps/details?id=org.fossify.home)   |
|  **Music Player**  |      [✔️](https://play.google.com/apps/testing/org.fossify.musicplayer)       |     ✔️      |     ✔️      |  [Download](https://play.google.com/store/apps/details?id=org.fossify.musicplayer)  |
|     **Notes**      |      [✔️](https://play.google.com/apps/testing/org.fossify.notes)       |     ✔️      |     ❌      |     [Download](https://play.google.com/store/apps/details?id=org.fossify.notes)     |
|      **Paint**      |      [✔️](https://play.google.com/apps/testing/org.fossify.paint)       |     ➖      |     ❌      |     [Download](https://play.google.com/store/apps/details?id=org.fossify.paint)      |
| **SMS Messenger**  |      [✔️](https://play.google.com/apps/testing/org.fossify.messages)       |     ✔️      |     ✔️      |   [Download](https://play.google.com/store/apps/details?id=org.fossify.messages)    |
| **Voice Recorder** |      [✔️](https://play.google.com/apps/testing/org.fossify.voicerecorder)       |     ✔️      |     ✔️      | [Download](https://play.google.com/store/apps/details?id=org.fossify.voicerecorder) |
|   **PDF Viewer**   |      ❌       |     ❌      |     ❌      |   [Download](https://play.google.com/store/apps/details?id=org.fossify.documents)   |
|   **Thank You**   |      [✔️](https://play.google.com/apps/testing/org.fossify.thankyou)       |      ✔️     |     ❌      |   [Download](https://play.google.com/store/apps/details?id=org.fossify.thankyou)   |


**Closed Track:** Only available to internal testers i.e. people who sent their email to hello@fossify.org. You must use the link above.
**Open Track:** Available to everyone but with an early access warning. Reviews are private.
**Production:** Available to everyone. Reviews are public.

_Comment inspired by [this.](https://github.com/orgs/FossifyOrg/discussions/9#discussioncomment-8094221)_

**Reply by nehemiagurl** [reply]
yo! thanks for making this!

**Reply by GitCurious** [reply]
Excellent - thankyou !

**Reply by naveensingh** [reply]
**Update:** Fossify Contacts is now available for testing. Join on [Android](https://play.google.com/store/apps/details?id=org.fossify.contacts) or the [web](https://play.google.com/apps/testing/org.fossify.contacts).

**Reply by GitCurious** [reply]
Will closed track test versions **automatically** update to production versions on GPlay once they are finalised ?

**Reply by naveensingh** [reply]
@GitCurious No, everything is reviewed by Google (for good reasons). Once 14 days have passed, I'll have to fill out a form for production access and once that is granted I have to issue an update which will then be reviewed and published automatically.

These testing requirements didn't exist before. Before, you could publish an app directly on the production channel as soon as it was reviewed.

**Reply by GitCurious** [reply]
Thanks for the info

**Reply by GuitarBilly** [reply]
@naveensingh can you elaborate a bit on what is needed to get Notes out of closed track and into the Play Store? 
More insight can help the beta-testers get things moving.

**Reply by naveensingh** [reply]
We have enough testers. Google Play flagged the notes app as misleading. I'm working on re-releasing it.

**Comment by realeatham** [comment]
I have sent an email

**Comment by naveensingh** [comment]
UPDATE: Fossify Phone is now available for testing. Join on [Android](https://play.google.com/store/apps/details?id=org.fossify.phone) or the [web](https://play.google.com/apps/testing/org.fossify.phone).

**Comment by Doehni** [comment]
Another Email sent :)

**Comment by naveensingh** [comment]
**Update:** 
Fossify File Manager is the first Fossify app that is now publicly available on the Google Play Store. It's a little too behind in the search results so here's the link again: https://play.google.com/store/apps/details?id=org.fossify.filemanager

The gallery is still being reviewed for closed testing (apparently). I contacted the support team about it and they responded with this:

> Our team is still investigating the most efficient and secure way to resolve the issue you reported.

**Reply by ronniedroid** [reply]
Second search result on aurora store. 

Good job everyone.

**Reply by naveensingh** [reply]
Fossify Calendar is the second app that is now publicly available on the Google Play Store. Thanks guys!

**Comment by flokoe** [comment]
Hi there, is there anything we can do to advance those apps in closed track (e.g., more testers) or do we just need to wait? I am not really familiar with how Google handles these publishing life cycles.

**Reply by naveensingh** [reply]
@flokoe More testers will help as not everyone is installing every Fossify app. We need  20+ active testers for at least 14 days and then we may request production access. So far we have 134 testers.

**Comment by naveensingh** [comment]
**Fossify Voice Recorder** is now available for internal testing (approved in under 12 hours). Please create a bug report if you encounter any issues.

Meanwhile, **Fossify Gallery** is still stuck in review and so far I haven't received any helpful answers from the support team.

**Comment by domestomas** [comment]
E-mail send to hello@fossify.org

**Comment by naveensingh** [comment]
### **Updates:**

**[Fossify Gallery](https://play.google.com/store/apps/details?id=org.fossify.gallery)** is finally available for closed testing! I'm yet to add some of the new emails so don't worry if some of you can't install it yet, I'll be sending an email to everyone who sent their email to mailto:hello@fossify.org. 

**[Fossify Music Player](https://play.google.com/store/apps/details?id=org.fossify.musicplayer)** is now publicly available on Google Play! 

**Fossify Voice Recorder** is now being reviewed for production and should soon be publicly available.

**Reply by naveensingh** [reply]
**[Voice Recorder](https://play.google.com/store/apps/details?id=org.fossify.voicerecorder)** is now available to everyone.

**Reply by GuitarBilly** [reply]
@naveensingh can you update above table for the general public? I find it a very useful overview including links for the overall Fossify status.

**Reply by naveensingh** [reply]
@GuitarBilly done!

**Comment by GuitarBilly** [comment]
email sent 3rd of March, would appreciate a reply.

**Reply by naveensingh** [reply]
@GuitarBilly I haven't received any emails on 3rd

**Reply by GuitarBilly** [reply]
@naveensingh  I was waiting for a reply on my volny mail where I sent the request...
But you already processed my gmail address as I got your invite.  Appreciated !

p.s. I could not see Voice Recorder in Play Store but installed it via the closed-track link in the table. should I track this page for new apps or can i watch the Play Store?

**Reply by naveensingh** [reply]
@GuitarBilly Voice Recorder is still being reviewed for production. 

> should I track this page for new apps or can i watch the Play Store?

I'll email you when a new app is released.

**Comment by Asiaboy89** [comment]
Looking forward to having the Notes App in the Play Store ;)

**Reply by naveensingh** [reply]
I'll try to do it by the weekend.

**Comment by tenextractor** [comment]
What is the timeline for Keyboard?

**Reply by naveensingh** [reply]
It is the next one, I don't have any ETAs.

**Comment by WebworkrNet** [comment]
Do you need additional testers?

**Reply by naveensingh** [reply]
I think we have enough testers now.

**Comment by himanshu-mahendra-jain** [comment]
@naveensingh Please check mail about swap.

**Reply by naveensingh** [reply]
@hj-collab done!

**Comment by naveensingh** [comment]
After months of back and forth with the Google Play team, **[Fossify Gallery](https://play.google.com/store/apps/details?id=org.fossify.gallery) is finally available for everyone.** Thank you!

**Comment by w3teal** [comment]
Hello, @naveensingh! Are i still be able to become tester too? Because the last comment is in about 3 months ago. I was a fan of the old Simple Notes. 
This is my email : ~~_deleted_~~
Thanks! :)

**Reply by naveensingh** [reply]
Added you to the testers list :+1:

**Comment by naveensingh** [comment]
**Update:**

Fossify Notes is now publicly available on Google Play: https://play.google.com/store/apps/details?id=org.fossify.notes

Fossify Keyboard is available to registered testers only: https://play.google.com/store/apps/details?id=org.fossify.keyboard

**Comment by naveensingh** [comment]
**Update:** 

**Fossify Launcher** and **Fossify Thank You** now available to registered testers:

-  https://play.google.com/store/apps/details?id=org.fossify.home
-  https://play.google.com/store/apps/details?id=org.fossify.thankyou

**Reply by realeatham** [reply]
Nice, thank you!

**Comment by naveensingh** [comment]
**Fossify Camera** is now available to registered testers: https://play.google.com/apps/testing/org.fossify.camera

**Reply by GuitarBilly** [reply]
hello @naveensingh, the latests apps Launcher, Thank You and Camera are incompatible with my phone. 
Did you change the requirement for Android? ( yes I have an ancient phone with Android 7 )

**Reply by Aga-C** [reply]
@GuitarBilly 
https://github.com/orgs/FossifyOrg/discussions/241

**Comment by naveensingh** [comment]
### Google hasn't been nice to Fossify:

**Fossify Calculator** was just **suspended** by Google with "repeated app rejections due to misleading claims" as the reason. They say the installed icon, title differ from what is displayed on the Google Play Store. I submitted an appeal but it's highly likely that it won't be reinstated.
  <details> 

  <summary>Click for more info regarding Fossify Calculator</summary>

![image](https://github.com/user-attachments/assets/208ae32d-8c41-4294-b9f8-1c76359c6a41)

The misleading icon and title as generated/provided by Google:

![LAUNCHER_ICON-2959](https://github.com/user-attachments/assets/0bfc980a-6a56-4ae3-9b33-6be8a1d55a1b)


  </details>

---

**Fossify Notes** was also rejected with weird reasons but they accepted it after I submitted an appeal.

  <details> 

  <summary>Click to see potential proof of Google using bots to review apps</summary>

![image](https://github.com/user-attachments/assets/518d76d0-01de-422f-9c80-35929ccaed67)

Misleading in-app content as generated/provided by Google:

<img alt="IN_APP_EXPERIENCE-2276" src="https://github.com/user-attachments/assets/9088b4e4-0a4a-406a-b686-1e4f4937b686" width=264 /><img alt="APP_SCREENSHOT-1962" src="https://github.com/user-attachments/assets/bc361c27-2eda-45f2-b9ba-0b3f83e91e8b" width=264/>

  <img alt="IN_APP_EXPERIENCE-3663" src="https://github.com/user-attachments/assets/71fa595e-66da-4529-8f10-c13976d92c06" width="528" />  <img alt="APP_SCREENSHOT-5214" src="https://github.com/user-attachments/assets/3adf364b-8785-4fbd-8b89-a3c66a9542f2" width="264" /> 

  </details>

---

The "review team" somehow thinks **Fossify Launcher** is collecting user's phone number and transmitting it off device but that doesn't seem possible given the permissions requested by the app and the fact that it can't even access the internet. This is another reason to think Google is relying on bots now.

  <details>

  <summary>Click to see more info regarding Fossify Launcher</summary>

  ![image](https://github.com/user-attachments/assets/e1f58183-b18a-4fbc-9102-474747e066f7)

  </details>

To be clear, I don't have anything against bots and automated systems but they should be reliable.

**Reply by PeterKW** [reply]
They will still be available to download on FDroid?

https://f-droid.org/packages/org.fossify.calculator/
https://f-droid.org/packages/org.fossify.notes/
https://f-droid.org/packages/org.fossify.home/

**Reply by naveensingh** [reply]
Yeah, absolutely. So far only Calculator has been removed from Google Play and I'll re-release it under a new package name if they don't change their mind about it.

**Reply by ronniedroid** [reply]
Oh wow, I wonder how all the scam shitty apps gets on the play store.

**Reply by naveensingh** [reply]
Fossify Calculator will be re-released: https://github.com/FossifyOrg/Calculator/discussions/36

**Reply by naveensingh** [reply]
Fossify Calculator is now released, available to registered testers only:

- **Web:** https://play.google.com/apps/testing/org.fossify.math
- **Android:** https://play.google.com/store/apps/details?id=org.fossify.math

**Reply by kylagit** [reply]
They probably don't like it because the Fossify Calc icon can be confused for Samsung's old calculator icon.
![fe2hj5bw9ms91](https://github.com/user-attachments/assets/be28aff7-f6ba-4759-b3c1-fc67f9b8cdae)

**Reply by naveensingh** [reply]
> Android: [play.google.com/store/apps/details?id=org.fossify.math](https://play.google.com/store/apps/details?id=org.fossify.math)

It's up now.

**Reply by kylagit** [reply]
Thank you. Though if you don't want it to come back down, I'd recommend changing the name in the app drawer to "Fossify Calc" and making the icon another colour such as black out of the box, with settings to let the user change it if they want.

**Reply by PytatoDuck** [reply]
I have been working on making and publishing my first app on Play Store, and I also faced the same issue. It's baffling to see how a Trillion-dollar company can make such a bad system. And I understand your frustration on this, as a literal brick would give you more feedback than Google Support.

I love Fossify, and really believe in this project. I hope a Media outlet picks up on this, as I don't see Google fixing this mess on their own.

**Comment by jidanni** [comment]
Please put the  link to [all Fossify Google Play Store apps](https://play.google.com/store/apps/dev?id=7297838378654322558) onto https://github.com/FossifyOrg .

**Comment by naveensingh** [comment]
**[Fossify Documents](https://www.fossify.org/apps/documents/)** is now available. See https://github.com/orgs/FossifyOrg/discussions/906 for more info.
